tasks/main.yml

```
@@ -161,6 +173,12 @@
     dest: "{{ ansible_user_dir }}/.config/systemd/user/{{ ya_provider_unit_name }}.service"
   notify: Restart provider
 
+- name: Add installer resources
+  ansible.builtin.shell:
```

we're not there yet (but hopefully soon)
```suggestion
  shell:
```